core/shared/src/main/scala/spire/std/sortedMap.scala

```diff
@@ -0,0 +1,141 @@
+package spire
```

Comment:
The `seq` and `map` instances in Spire are for the generic base type. Here we introduce a specialization for `SortedMap`.

What is the future of this design ? move to higher-kinded instances (often conflicts with specialization) ? add special cases as needed ?